### PR TITLE
Preparing relase 1.21.0

### PR DESCRIPTION
**Pull request by rubenvp8510**
Signed-off-by: Ruben Vargas Palma <ruben.vp8510@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,44 @@
 Changes by Version
 ==================
 
+1.21.0 (2020-11-17)
+-------------------
+
+* Regenerate self-provisioned ES TLS cert when it's outdated ([#1301](https://github.com/jaegertracing/jaeger-operator/pull/1301), [@kevinearls](https://github.com/kevinearls))
+* Update the TestCreateElasticsearchCR unit test to include a case with… ([#1299](https://github.com/jaegertracing/jaeger-operator/pull/1299), [@kevinearls](https://github.com/kevinearls))
```
Comment (objectiser) [inline CHANGELOG.md:8]: PRs related to tests or internal CI should be removed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,44 @@
...
+* Enable tolerations support in elasticsearch config ([#1296](https://github.com/jaegertracing/jaeger-operator/pull/1296), [@kevinearls](https://github.com/kevinearls))
+* Update github.com/miekg/dns to v1.1.35 ([#1298](https://github.com/jaegertracing/jaeger-operator/pull/1298), [@objectiser](https://github.com/objectiser))
+* Add serviceType for the collector service ([#1286](https://github.com/jaegertracing/jaeger-operator/pull/1286), [@sschne](https://github.com/sschne))
+* Add env var JAEGER_DISABLED ([#1285](https://github.com/jaegertracing/jaeger-operator/pull/1285), [@sschne](https://github.com/sschne))
+* Fix secret creation when using self provisioned elasticsearch instances ([#1288](https://github.com/jaegertracing/jaeger-operator/pull/1288), [@kevinearls](https://github.com/kevinearls))
+* Remove unused start-openshift.sh script ([#1293](https://github.com/jaegertracing/jaeger-operator/pull/1293), [@objectiser](https://github.com/objectiser))
```
Comment (objectiser) [inline CHANGELOG.md:14]: Remove from list

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,44 @@
...
+* Convert storage type to typed string ([#1282](https://github.com/jaegertracing/jaeger-operator/pull/1282), [@SezalAgrawal](https://github.com/SezalAgrawal))
+* Use New Admin Port Flag ([#1281](https://github.com/jaegertracing/jaeger-operator/pull/1281), [@johanavril](https://github.com/johanavril))
+* Skip generator test until issue 1278 if resolved ([#1279](https://github.com/jaegertracing/jaeger-operator/pull/1279), [@kevinearls](https://github.com/kevinearls))
```
Comment (objectiser) [inline CHANGELOG.md:17]: remove

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,44 @@
...
+* Update instances status using client.Status().update interface ([#1253](https://github.com/jaegertracing/jaeger-operator/pull/1253), [@rubenvp8510](https://github.com/rubenvp8510))
+* Update openshift only tests with new location of examples ([#1274](https://github.com/jaegertracing/jaeger-operator/pull/1274), [@kevinearls](https://github.com/kevinearls))
```
Comment (objectiser) [inline CHANGELOG.md:19]: remove

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,44 @@
...
+* Remove gRPC host-port from being added to the CR (agent) ([#1272](https://github.com/jaegertracing/jaeger-operator/pull/1272), [@jpkrohling](https://github.com/jpkrohling))
+* Sync OTEL config volume/mount and args ([#1268](https://github.com/jaegertracing/jaeger-operator/pull/1268), [@jpkrohling](https://github.com/jpkrohling))
+* Fix csv generation in release script ([#1247](https://github.com/jaegertracing/jaeger-operator/pull/1247), [@rubenvp8510](https://github.com/rubenvp8510))
```
Comment (objectiser) [inline CHANGELOG.md:22]: remove

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,44 @@
...
+* Publish container - dockerx should not use tag BUILD_IMAGE ([#1270](https://github.com/jaegertracing/jaeger-operator/pull/1270), [@morlay](https://github.com/morlay))
```
Comment (objectiser) [inline CHANGELOG.md:23]: Even though this (and the next) are related to internal builds, this user has provided useful contributions, so should be recognized in the changelog.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,6 +1,44 @@
...
+* Speed up buildx process ([#1267](https://github.com/jaegertracing/jaeger-operator/pull/1267), [@morlay](https://github.com/morlay))
+* Set mergify to dismiss stale reviews ([#1269](https://github.com/jaegertracing/jaeger-operator/pull/1269), [@jpkrohling](https://github.com/jpkrohling))
```
Comment (objectiser) [inline CHANGELOG.md:25]: remove

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,6 +1,44 @@
...
+* Fix the dependencies ([#1264](https://github.com/jaegertracing/jaeger-operator/pull/1264), [@faceair](https://github.com/faceair))
+* Add agent hostNetwork option ([#1257](https://github.com/jaegertracing/jaeger-operator/pull/1257), [@faceair](https://github.com/faceair))
+* Skip detectClusterRoles for Kubernetes ([#1262](https://github.com/jaegertracing/jaeger-operator/pull/1262), [@johanavril](https://github.com/johanavril))
+* Elasticsearch: add SYS_CHROOT capability ([#1260](https://github.com/jaegertracing/jaeger-operator/pull/1260), [@haircommander](https://github.com/haircommander))
+* Allow overriding the vertx example app image and config values ([#1259](https://github.com/jaegertracing/jaeger-operator/pull/1259), [@kevinearls](https://github.com/kevinearls))
+* Simplify OTEL related environment variables ([#1255](https://github.com/jaegertracing/jaeger-operator/pull/1255), [@kevinearls](https://github.com/kevinearls))
+* Add tests for OTEL ingester ([#1252](https://github.com/jaegertracing/jaeger-operator/pull/1252), [@kevinearls](https://github.com/kevinearls))
```
Comment (objectiser) [inline CHANGELOG.md:32]: remove

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,6 +1,44 @@
...
+* Replaced devbots with issue-labeler ([#1251](https://github.com/jaegertracing/jaeger-operator/pull/1251), [@jpkrohling](https://github.com/jpkrohling))
```
Comment (objectiser) [inline CHANGELOG.md:33]: remove

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,6 +1,44 @@
...
+* Add CQLSH_PORT environment variable ([#1243](https://github.com/jaegertracing/jaeger-operator/pull/1243), [@Ashmita152](https://github.com/Ashmita152))
+* expose elasticsearch container ports ([#1224](https://github.com/jaegertracing/jaeger-operator/pull/1224), [@jkandasa](https://github.com/jkandasa))
+* Add autoscale test for ingester ([#1233](https://github.com/jaegertracing/jaeger-operator/pull/1233), [@kevinearls](https://github.com/kevinearls))
```
Comment (objectiser) [inline CHANGELOG.md:36]: remove

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,6 +1,44 @@
...
+* Updated the Operator Hub PR template ([#1234](https://github.com/jaegertracing/jaeger-operator/pull/1234), [@jpkrohling](https://github.com/jpkrohling))
```
Comment (objectiser) [inline CHANGELOG.md:37]: remove

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,6 +1,44 @@
...
+* Adjusted OLM manifests for 1.20, adjusted release script ([#1226](https://github.com/jaegertracing/jaeger-operator/pull/1226), [@jpkrohling](https://github.com/jpkrohling))
```
Comment (objectiser) [inline CHANGELOG.md:38]: remove

**Review by objectiser — COMMENTED** [review]
Seems to be a problem with the tests - hopefully when you update the changelog it will go green.

**Comment by rubenvp8510** [comment]
> Seems to be a problem with the tests - hopefully when you update the changelog it will go green.

It's a problem with the actions, I think we need to update that action.

 https://github.com/jpkrohling/setup-operator-sdk/pull/5

File: CHANGELOG.md (hunk 6 above)
Comment (rubenvp8510) [reply CHANGELOG.md:18]: +1

**Comment by objectiser** [comment]
@rubenvp8510 looks like the action has been released (v1.0.1) - but https://github.com/jaegertracing/jaeger-operator/blob/master/.github/workflows/release.yaml#L15 seems to be referencing `v1-release` - so thought may be it was referencing a docker image.

**Comment by jpkrohling** [comment]
You can change v1-release to v1.0.1 and it should use the latest release.

**Comment by rubenvp8510** [comment]
@jpkrohling action is still failing :(  with:
```
internal/modules/cjs/loader.js:800
    throw err;
    ^

Error: Cannot find module '@actions/core'
Require stack:
- /home/runner/work/_actions/jpkrohling/setup-operator-sdk/v1.0.1/lib/main.js
```

Also I think https://github.com/jpkrohling/setup-kubectl requires the same update :)

**Comment by jpkrohling** [comment]
> Also I think https://github.com/jpkrohling/setup-kubectl requires the same update :)

I also released this one. As for the other failure, I'm not sure what's the cause. Is this something I have to change in the action?

**Comment by jpkrohling** [comment]
Looks like the new versions of javascript actions need node_modules checked in. I did that in the latest master: could you try using that? Replace `@v1.0.1` with `@master`.

**Review by objectiser — APPROVED** [review]

**Pull request merged**